README.md

```diff
@@ -9,8 +9,6 @@ Itemizer is a Sponge Minecraft plugin that allows custom item creation. Store pl
 
 An examples of each config file is loaded at the first launch of the plugin under the folder `/config/itemizer/` you can see how they work.
```

Comment:
`An example` sans S  

La fin de ta phrase, les deux groupes peuvent pas aller ensemble sans separation: `an example [...] under the folder...` et `you can see how they work`. Si tu lis tout d'un coup tu dois comprendre le souci. Met un point apres le path de ton folder. Pour faire du plus bel anglais, tu peux aussi remplacer la suite par `This way, you can see how they're built.` (ou ce que tu veux, mais pas juste un bout de phrase `tu peux voir comment ca marche` place au milieu de nulle part)